apps/web/src/content/types/Blog.ts

```diff
@@ -0,0 +1,58 @@
+import { defineDocumentType } from "contentlayer/source-files";
+import readingTime from "reading-time";
+
+export const Blog = defineDocumentType(() => ({

```

Comment:
Let's rename to `Post` (and the corresponding names).

Reply:
Alright